src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure implements AreaCalculetion {
+    private double side;
+
+    public Square(double side, String color, String name) {

```

Comment:
The constructor of 'Square' does not call the superclass constructor to set the color and name. It should call 'super(color, name)' to properly initialize these properties inherited from 'Figure'.